data/web/js/site/user.js

```diff
@@ -224,6 +225,11 @@ jQuery(function($){
             if (!item.last_run > 0) {
               item.last_run = lang.waiting;
             }
+            if (item.success == null) {
+              item.success = '-';

```

Comment:
Should this be "currently running" or similar?

Reply:
Hm, we have is_running, so that's ok, I guess.

Reply:
Hi, the "item.success" is set to null when a job enters running state or when a job is rescheduled.